src/topp/MzTabExporter.cpp

```diff
@@ -782,8 +782,7 @@ namespace OpenMS
         db_version = sp.db_version.empty() ? MzTabString() : MzTabString(sp.db_version);
       }
 
-      // currently we don't save how many channels we saved in a consensus feature so we scan the file
-      // for the maximum map index in a consensus feature and take this as number of study variables
+      // determine number of channels
       Size n_study_variables = consensus_map.getFileDescriptions().size();
```

Comment:
This works for label-free, but not for labelled datasets. In the labelled case, you can have one file but three channels.